relayer/message_relayer.go

```diff
@@ -53,17 +54,20 @@ type messageRelayer struct {
 	relayer                 *Relayer
 	warpMessage             *avalancheWarp.UnsignedMessage
 	destinationBlockchainID ids.ID
+	warpQuorum              config.WarpQuorum
 }
 
 func newMessageRelayer(
 	relayer *Relayer,
 	warpMessage *avalancheWarp.UnsignedMessage,
 	destinationBlockchainID ids.ID,
+	cfg config.Config,
```

Comment:
Let's just make the config a field in the `Relayer` struct so we don't have to pass it around like this

Reply:
Good call, made this change.